mscore/musescoredialogs.cpp

```diff
@@ -158,6 +158,25 @@ AboutBoxDialog::AboutBoxDialog()
       revisionLabel->setText(tr("Revision: %1").arg(revision));
       setWindowFlags(this->windowFlags() & ~Qt::WindowContextHelpButtonHint);
 
+      auto compilerDateISO = []() -> std::string {
+            // Store __DATE__ in ISO 8601 format ( e.g. 2025-04-20 )
+            int month, day, year;
+            char buffer[10];
+            static const char monthNames[] = "JanFebMarAprMayJunJulAugSepOctNovDec";
+            sscanf(__DATE__, "%s %d %d", buffer, &day, &year);
+            month = ((strstr(monthNames, buffer) - monthNames) / 3) + 1;
+            sprintf(buffer, "%d-%02d-%02d", year, month, day);
+            return std::string(buffer);
+            };
+
+      bool useISOFormat = false;
```

Comment:
P.S. I didn't want to delete that little lambda for the ISO version, so I'll do a ternary here if that's okay with you